client/my-sites/plugins/main.jsx

```diff
@@ -426,7 +425,7 @@ const PluginsMain = React.createClass( {
 export default connect(
 	state => {
 		return {
-			wporgPlugins: state.plugins.wporg
+			plugins: state.plugins.wporg
```

Comment:
Also, this is the main reason behind having a `/state/plugins/wporg/` subfolder. Right now the only part that is migrated to redux is the store that deals with .org plugins info. The part that deals with user plugins info is still on a flux store (https://github.com/Automattic/wp-calypso/blob/master/client/lib/plugins/store.js) which would need to be migrated to redux too, so we probably would end with a `/state/plugins/sites` or something like that too